### PR TITLE
Fix availability

### PR DESCRIPTION
**Pull request by chanteriam**
## Describe your changes
This PR adds day of week and time of day to the availability table and removes the calendar table. This results in faster update/creation time for user availability (from 50 to 5 seconds)

## Checklist before merging

- [x] My code produces no errors when run
- [x] My code is linted and properly formatted

### CODE REVIEW

**Review by sarahwalker10 — CHANGES_REQUESTED** [review]
**2 changes, 1 question:**

Change 1. Should the playground.py and playground2.py files be removed? They seem like testing files.

Change 2. The transition from onboarding to seeing the completed profile is still missing (it remains on this page indefinitely):

Screenshot 2024-05-08 at 12 26 25 PM



Question 1. Also quick question just bc I'm curious, why this is like this in the availability.py model (tuples with the repeated values)?:
```
DAY_CHOICES = (
        ('Monday', 'Monday'),
        ('Tuesday', 'Tuesday'),
        ('Wednesday', 'Wednesday'),
        ('Thursday', 'Thursday'),
        ('Friday', 'Friday'),
        ('Saturday', 'Saturday'),
        ('Sunday', 'Sunday'),
    ) 
```

**Comment by sarahwalker10** [comment]
However, can confirm that 1. creating the profile and having the info saved to the database happens in only a few seconds; and 2. updating the availability and making profile updates get changed in the database also very quickly (whereas previously it took over 1 min to update calendar availability) (P.S. nice touch adding the pop-up "Please wait while we update your availability")

File: playground.py
Comment (ehabich) [inline playground.py:1]: Agree with sarah on this file. Is it mean tot be included? It seems like yes but would you change the filename to somethign more informative if so or delete if not?

File: playground2.py
Comment (ehabich) [inline playground2.py:1]: same on this file

**Review by ehabich — CHANGES_REQUESTED** [review]

**Review by sarahwalker10 — APPROVED** [review]

**Review by sarahwalker10 — DISMISSED** [review]

**Pull request merged**